tasks/install.yml

```
@@ -40,7 +28,9 @@
         enabled: true
         state: started
   when:
-    - ansible_lsb.release is version('20.04', '<')
+    - |
+      ansible_lsb.release is defined and 
+      ansible_lsb.release is version('20.04', '<')
```

```suggestion
    - ansible_lsb.release is defined 
    - ansible_lsb.release is version('20.04', '<')
```
If you use lists for `when` then they're `AND` by default anyway.